src/UI/ActionButtons.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package UI;
+
+import java.awt.event.ActionEvent;
+import javax.swing.JButton;
+import javax.swing.JPanel;
+/**
+ *
+ * @author Shirley Pinto
+ */
+public class ActionButtons extends JPanel {
+    private JButton jButtonCopy;
+    private JButton jButtonDelete;
+    private JButton jButtonEdit;
+    private JButton jButtonMove;
+    
+    public ActionButtons() {
+        jButtonEdit = new javax.swing.JButton();
```

Comment:
you dont need to use the full package for the Jbutton since you already imported that package